src/test/feature/README.md

```diff
@@ -15,8 +15,7 @@ Before building the code of feature tests part, just make sure your compiler sup
 1. Make sure HAWQ is running correctly. If not, `init` or `start` HAWQ at first. Note please don't set locale related arguments for hawq init.
 2. Load environment configuration by running `source $INSTALL_PREFIX/greenplum_path.sh`.
 3. Load hdfs configuration. For example, `export HADOOP_HOME=/Users/wuhong/hadoop-2.7.2 && export PATH=${PATH}:${HADOOP_HOME}/bin`. Since some test cases need `hdfs` and `hadoop` command, just ensure these commands work before running. Otherwise you will get failure.
-4. Run the cases with`./parallel-run-feature-test.sh 8 ./feature-test`(in this case 8 threads in parallel), you could use `--gtest_filter` option to filter test cases(both positive and negative patterns are supported). Please see more options by running `./feature-test --help`.
-5. As for now, there are several cases which in the sequence.txt file can not parallel run. Run the cases with `./parallel-run-feature-test.sh 8 ./feature-test --gtest_filter=-`cat ./sequence.txt``.
+4. Run the cases with`./parallel-run-feature-test.sh 8 ./feature-test`(in this case 8 threads in parallel), you could use `--gtest_filter` option to filter test cases(both positive and negative patterns are supported). You can also run cases with `--gtest_schedule` (eg. --gtest_schedule=./schedule.txt) if you want to run cases in both parallel way and serial way. Please see more options by running `./feature-test --help`.
```

Comment:
Already added an example in the README.md , so can not get your points. Please be more specific , thank you